src/app/config.ts

```diff
@@ -19,7 +22,8 @@ const env = cleanEnv(import.meta.env, {
   VITE_KEYCLOAK_REALM: str(),
   VITE_KEYCLOAK_CLIENT_ID: str(),
   VITE_API_BASE_URL: url(),
-  VITE_FORECAST_USER_ID_OVERRIDE: num({ default: undefined })
+  VITE_FORECAST_USER_ID_OVERRIDE: num({ default: undefined }),
+  VITE_PIPEDRIVE_URL: url()
```

Comment:
This should be named accordingly for the home lambdas url, it is not a pipedrive url which is a seperate service not currently in use here